src/main/java/Astra.java

```diff
@@ -0,0 +1,101 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    public void markAsNotDone() {
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // "X" for done, " " for not done
```

Comment:
I like the clear inline comment :)
```suggestion
        return (isDone ? "X" : " "); // "X" for done, " " for not done
```